client/logic/globals/types.gd

```diff
@@ -69,9 +69,9 @@ func _generate_types() -> void:
 
 
 func add_text_to_gdscript(
-	file: FileAccess, name: String, text: String, context: String = ""
+	file: FileAccess, var_name: String, text: String, context: String = ""
```

Comment:
Var_name? Can we find a better name?

Reply:
I changed it because of shadowing warning. How would you call it? My thought was, it's a name for a variable.

Reply:
String_id ? Or string name?

Reply:
It is now string_name.